OsmAnd/src/net/osmand/plus/settings/fragments/GeneralProfileSettingsFragment.java

```diff
@@ -298,6 +295,33 @@ private void setupExternalInputDevicePref() {
 		}
 	}
 
+	private void setupPositionAnimation() {
+		Preference uiPreference = findPreference(settings.ANIMATE_MY_LOCATION.getId());
+
+		if (uiPreference != null) {
+			int interpolationValue = settings.LOCATION_INTERPOLATION_PERCENT.get();
+			boolean animationEnabled = settings.ANIMATE_MY_LOCATION.get();
```

Comment:
if app mode is car & we open preferences for bicycle, will this get work properly or we need getModeValue?